hazelcast-sql/pom.xml

```diff
@@ -176,7 +176,9 @@
                                     <include>org.apache.calcite:calcite-linq4j</include>
                                     <include>org.apache.calcite.avatica:avatica-core</include>
                                     <include>org.checkerframework:checker-qual</include>
-                                    <include>com.jayway.jsonpath:json-path</include>
+                                    <include>com.hazelcast.jsurfer:jsurfer-core</include>
+                                    <include>com.hazelcast.jsurfer:jsurfer-jackson-jr</include>
+                                    <include>org.antlr:antlr4-runtime</include>
```

Comment:
Do you want to relocate the antlr under `com.hazelcast.org.antlr` or similar? It's done for all 3rd party deps in sql module.

Reply:
Should be shaded now.

Reply:
`hazelcast-sql` depends on `JsonSurfer` which depends on `antlr-runtime`. Will classes from `JsonSurfer` be aware of such relocation?

Reply:
I checked, works as expected. `JsonSurfer` classes use shaded `antlr-runtime`.